src/app/shared/item-card/item-card.component.ts

```diff
@@ -12,10 +18,31 @@ export class ItemCardComponent implements OnInit {
   @Input() item: Item;
 
   private mobileWidth: number;
+  private ready: boolean = false;
+  private basePath:string = '/uploads';
 
-  constructor(private windowService: WindowService) {}
+  constructor(
+    private windowService: WindowService,
+    private dataService: DataService,
+    private db: AngularFireDatabase) { }
 
   ngOnInit() {
     this.mobileWidth = MOBILE_WIDTH;
+
+    var id = String(localStorage.getItem('item' + this.item.id));
```

Comment:
Are we storing our items in localStorage? I thought we're retrieving from firebase.

Reply:
Since there is not yet the ability to store the Firebase url for the image in our database and backend I am temporarily adding the url's to local storage. I will create a separate issue for this.